test/logs/ARM_MUSCA_S1/STORAGE.log

```diff
@@ -0,0 +1,6 @@
+Storage Suite Report
+TOTAL TESTS     : [0-9]+
+TOTAL PASSED    : [0-9]+
+TOTAL SIM ERROR : 0
+TOTAL FAILED    : [0-1]
```

Comment:
[Musca_S1_GNUARM_STORAGE.log](https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/files/5224056/GNUARM_STORAGE.log) refers that there are no failed tests, is there a reason one test failed is to be considered here?

Reply:
This must have been a typo, it is changed.